doozerlib/cli/__main__.py

```diff
@@ -2059,7 +2063,18 @@ def beta_reposync(runtime, output, cachedir, arch, repo_type, names, dry_run):
                 continue
 
             color_print('Syncing repo {}'.format(repo.name), 'blue')
-            cmd = cmd_base.format(yc_file.name, output, arch, repo.name, metadata_dir)
+            reposync_cmd = reposync_base.format(yc_file.name, output, arch, repo.name, metadata_dir)
+
+            cmd = 'podman run --rm --user {uid}:{gid} -v {tmpdir}:{tmpdir}:Z -v {output}:{output}:Z -v {metadata_dir}:{metadata_dir}:Z {image} {reposync_cmd}'.format(uid=os.getuid(),
+                                                                                                                                                                      gid=os.getgid(),
+                                                                                                                                                                      output=output,
+                                                                                                                                                                      tmpdir=tempdir_path,
+                                                                                                                                                                      metadata_dir=metadata_dir,
+                                                                                                                                                                      image=runtime_image,
+                                                                                                                                                                      reposync_cmd=reposync_cmd)
```

Comment:
could we not have the parameters all out on the right here...
```suggestion
cmd = 'podman run --rm --user {uid}:{gid} -v {tmpdir}:{tmpdir}:Z -v {output}:{output}:Z -v {metadata_dir}:{metadata_dir}:Z {image} {reposync_cmd}'.format(
    uid=os.getuid(),
    gid=os.getgid(),
    output=output,
    tmpdir=tempdir_path,
    metadata_dir=metadata_dir,
    image=runtime_image,
    reposync_cmd=reposync_cmd,
)
```